zmsdldb/src/Zmsdldb/Entity/Base.php

```
@@ -128,14 +130,20 @@ public static function doubleUnterlineToArray(&$array, string $key, $value)
+            }
             if (! isset($array[$key]) || ! is_array($array[$key])) {
                 $array[$key] = [];
             }
 
             $array = &$array[$key];
         }
 
-        $array[array_shift($keys)] = $value;
+        $lastKey = array_shift($keys);
+        if ($lastKey !== null) {
+            $array[$lastKey] = $value;
```

PHPStan (2.2.8)

[error] Parameter &$array by-ref type of method BO\Zmsdldb\Entity\Base::doubleUnterlineToArray() expects static(BO\Zmsdldb\Entity\Base), array|BO\Zmsdldb\Entity\Base given.
You can change the parameter out type with `@param-out` PHPDoc tag.

(parameterByRef.type)